spec/controllers/external_users/advocates/claims_controller_spec.rb

```diff
@@ -525,14 +525,14 @@ def valid_claim_fee_params
       'advocate_category' => 'QC',
       'offence_class_id' => '2',
       'offence_id' => offence.id.to_s,
-      'first_day_of_trial(3i)' => '13',
-      'first_day_of_trial(2i)' => '5',
-      'first_day_of_trial(1i)' => '2015',
+      'first_day_of_trial(3i)' => date.day,
+      'first_day_of_trial(2i)' => date.month,
+      'first_day_of_trial(1i)' => date.year,
       'estimated_trial_length' => '2',
       'actual_trial_length' => '2',
-      'trial_concluded_at(3i)' => '15',
-      'trial_concluded_at(2i)' => '05',
-      'trial_concluded_at(1i)' => '2015',
+      'trial_concluded_at(3i)' => date.day + 2,
```

Comment:
This will cause problems when `date.day` is within 2 days of the end of the month. It is safer to use another memoized variable:

```
let(:trial_concluded_date) { date + 2.days }
```